elasticsearch/kibana.sls

```diff
@@ -106,6 +92,23 @@ install_marvel:
   - unless: 'test -d /usr/share/kibana/installedPlugins/marvel'
```

Comment:
We'll probably want to add a `require_in` clause for 'file: /usr/share/kibana/optimize/.babelcache.json'